package-schema-registry/setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+from pip.req import parse_requirements
+import setuptools
+
+remote_requirements = '\n'.join(str(r.req) for r in parse_requirements("requirements.txt", session='dummy') if r.req)
+
+setuptools.setup(
+    name='cp-schema-registry-tests',
+    version='4.0.1',
```

Comment:
we might want to consider getting this programmatically from some other file. i'd hate to add yet another place we need to update version numbers.